apps/web/src/pages/_document.tsx

```diff
@@ -1,8 +1,11 @@
 import { Head, Html, Main, NextScript } from 'next/document'
+import Script from 'next/script'
 import { NextSeo } from 'next-seo'
 
 import { SERVICE_NAME } from '../constants'
 
+const GTM_ID = process.env.NEXT_PUBLIC_APP_ENV === 'prod' ? 'G-3ZXPW1HKG0' : 'G-YPS8HKL3BR'
```

Comment:
This needs to be configurable.

This isn't quite a secret, but it's close and it is environment specific, etc.